src/iotjs_debuglog.h

```diff
@@ -32,7 +32,7 @@ extern const char* iotjs_debug_prefix[4];
 #define IOTJS_DLOG(lvl, ...)                                        \
   do {                                                              \
     if (0 <= lvl && lvl <= iotjs_debug_level && iotjs_log_stream) { \
-      fprintf(iotjs_log_stream, "[%s] ", iotjs_debug_prefix[lvl]);  \
+      fprintf(iotjs_log_stream, "%s ", iotjs_debug_prefix[lvl]);    \
```

Comment:
maybe brackets are still needed as some tty may not supporting color rendering

Reply:
How to detect if a tty supports colors?

Reply:
Adding the brackets seems not resolving this problem, either. Because `COLOR_PRINT` still could not be proceed correctly.

Reply:
Did google out the `tput` as well, but I'm not going to put this tool on our system, that might be useless except us, another example is here: [chalk/supports-color](https://github.com/chalk/supports-color/blob/master/index.js), I saw it checks `env.TERM` and windows support.

So how about the following algorithm?

- check `isatty()`, if no, we just not print color any more.
- check `env.TERM` that follows [chalk/supports-color](https://github.com/chalk/supports-color/blob/master/index.js).
- windows terminal support.

And in this PR, we should only have the step 1 done is enough?

Reply:
LGTM